src/main/java/sg/ncl/MainController.java

```diff
@@ -2040,16 +2047,27 @@ public String joinTeamAppAwaitingApproval(Model model) {
 //		}
         // FIXME: hardcode list of filenames for now
         List<String> scenarioFileNameList = new ArrayList<>();
-        scenarioFileNameList.add("basic.ns");
-        scenarioFileNameList.add("basic2.ns");
+        scenarioFileNameList.add("Scenario 1 - A single node");
+        scenarioFileNameList.add("Scenario 2 - Two nodes linked with a 10Gbps link");
+        scenarioFileNameList.add("Scenario 3 - Three nodes in a star topology");
         logger.info("Scenario file list: {}", scenarioFileNameList);
 		return scenarioFileNameList;
     }
 
     private String getScenarioContentsFromFile(String scenarioFileName) throws WebServiceRuntimeException {
+        // FIXME: switch to better way of referencing scenario descriptions to actual filenames
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1134)